ocs/ocp.py

```diff
@@ -264,22 +267,24 @@ def get_ceph_tools_pod():
     return ct_pod
 
 
-def exec_ceph_cmd(ceph_cmd):
+def exec_ceph_cmd(ceph_cmd, format_arg='json-pretty'):
     """
     Execute a Ceph command on the Ceph tools pod
 
     Args:
         ceph_cmd (str): The Ceph command to execute on the Ceph tools pod
+        format_arg (str): The argument for format. Defaults to 'json-pretty'
 
     Returns:
-        dict: Ceph command output
+        dict: Ceph command output if the output is in Munch format
+        else, returns the output as a list
     """
     ocp_pod_obj = OCP(kind='pods', namespace=ENV_DATA['cluster_namespace'])
     ct_pod = get_ceph_tools_pod()
-    ceph_cmd += " --format json-pretty"
+    ceph_cmd += f" --format {format_arg}"
     out = ocp_pod_obj.exec_cmd_on_pod(ct_pod, ceph_cmd)
 
     # For some commands, like "ceph fs ls", the returned output is a list
-    if isinstance(out, list):
-        return [item.toDict() for item in out if item]
-    return out.toDict()
+    if isinstance(out, Munch):
+        return out.toDict()
+    return yaml.safe_load(out)
```

Comment:
```
return yaml.safe_load(out)
```
Is yaml.safe_load() still required? Here, 'out' will be a list

Reply:
out is a string here.